components/script/dom/document.rs

```diff
@@ -4655,6 +4655,60 @@ impl DocumentMethods<crate::DomTypeHolder> for Document {
         ))
     }
 
+    /// https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-parsehtmlunsafe
```

Comment:
You can catch such warnings with `./mach test-tidy`
```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-parsehtmlunsafe>
```